vllm/model_executor/layers/utils.py

```diff
@@ -100,12 +100,41 @@ def default_unquantized_gemm(
     return torch.nn.functional.linear(x, weight, bias)
 
 
+def use_aiter_triton_gemm(n, m, k, dtype):
+    if (
+        envs.VLLM_ROCM_USE_AITER == 0
+        or envs.VLLM_ROCM_USE_AITER_TRITON_GEMM == 0
```

Comment:
I believe the current approach for this is to enable TRITON_GEMM if we're on gfx950, and not worry about setting an explicit envvar